grakn-test/test-integration/src/test/java/ai/grakn/test/DistributionContext.java

```diff
@@ -78,10 +76,6 @@ public static DistributionContext startSingleQueueEngineProcess(){
         return new DistributionContext(RedisTaskManager.class);
```

Comment:
This can just be `start()` now.

In fact, I'd actually just call it `create()`: `start()` is kind of misleading... this doesn't start anything on its own :P